package.json

```diff
@@ -17,22 +17,22 @@
     "tiny-emitter": "^2.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.11.6",
-    "@babel/preset-env": "^7.11.5",
-    "babel-loader": "^8.1.0",
+    "@babel/core": "^7.12.10",
+    "@babel/preset-env": "^7.12.11",
+    "babel-loader": "^8.2.2",
     "chai": "^4.2.0",
-    "cross-env": "^5.2.0",
-    "karma": "^3.1.1",
+    "cross-env": "^7.0.3",
+    "karma": "^6.0.0",
     "karma-chai": "^0.1.0",
-    "karma-mocha": "^1.2.0",
-    "karma-chrome-launcher": "^2.2.0",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-mocha": "^2.0.1",
     "karma-sinon": "^1.0.4",
-    "karma-webpack": "^3.0.5",
-    "mocha": "^5.2.0",
-    "sinon": "^7.1.1",
-    "uglifyjs-webpack-plugin": "^2.0.1",
-    "webpack": "^4.5.0",
-    "webpack-cli": "^3.1.2"
+    "karma-webpack": "^5.0.0-alpha.5",
```

Comment:
Just to keep in mind to change this alpha version when having a v5 stable.

https://github.com/ryanclark/karma-webpack/releases/tag/v5.0.0-alpha.5